lib/byron/test/unit/Cardano/Wallet/Byron/NetworkSpec.hs

```diff
@@ -80,3 +82,8 @@ zeroTxParameters :: TxParameters
 zeroTxParameters = TxParameters
     (LinearFee (Quantity 0) (Quantity 0) (Quantity 0))
     (Quantity 0)
+
+-- | A value that is not the same as what's in the test data genesis.
+halfD :: DecentralizationLevel
+halfD = DecentralizationLevel p
+    where Right p = mkPercentage (1/2)
```

Comment:
It is sort of weird to have this unit test in the `byron` package; the decentralization level is something that exists only in Shelley.

Reply:
OK I moved it to the shelley package.